Cargo.toml

```diff
@@ -12,6 +14,7 @@ license = "MIT/Apache-2.0"
 
 [dependencies]
 rand = "0.4"
+remove_dir_all = "0.3"
```

Comment:
Looks like you can use `remove_dir_all` from stable Rust now. https://doc.rust-lang.org/std/fs/fn.remove_dir_all.html